sketch_oct17a.ino

```diff
@@ -2,10 +2,35 @@
 
 void setup() {
   // put your setup code here, to run once:
-
+  int motor1_pin = 7; //vertical 
+  int motor2_pin = 8; //horizontal
+  int claw_pin = 9;
+  int pos = 0;
+  SphericalGear sg(motor1_pin,motor2_pin);
+  Servo claw_servo;
+  claw_servo.attach(claw_pin); // initialize motor
+  //we want the arm to reach down, then grab with the claw, then pull back up
+  //we might want to go back down and release the claw as well
+  
+  // open claw
+  openclaw(claw_servo, 45);
```

Comment:
instead of using 45, maybe create a variable for the angle it opens. Call it something like "openAngle", so we can change it more easily to 30, 20, or whatever it needs to be.